package.json

```diff
@@ -15,6 +15,8 @@
     "@types/react": "^18.0.26",
     "@types/react-dom": "^18.0.9",
     "formik": "^2.2.9",
+    "moment": "^2.29.4",
```

Comment:
the moment is no longer a supported package, please use another for work with dates.

Reply:
done